althea/althea-rust-binaries/Makefile

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (C) 2017-2018 Justin Kilpatrick, Stanislaw Drozd, Openwrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=althea-rust-binaries
+PKG_VERSION:=0.1
+PKG_RELEASE:=0
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/althea-mesh/althea_rs
+PKG_SOURCE_VERSION:=9e7859c0db2208b66f7de7dd520924e4533df26d
+PKG_LICENSE:=Apache-2.0
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/althea-rust-binaries
+  SECTION:=net
+  CATEGORY:=utils
+  SUBMENU:=Utilities
+  TITLE:=Althea Rust executables
+  URL:=https://github.com/althea-mesh/althea_rs
+  MAINTAINER:= Justin Kilpatrick <justin@altheamesh.com>
+  DEPENDS:=@IPV6
+endef
+
+define Package/althea-rust-binaries/description
+  Althea Rust executables
+endef
+
+ifeq ($(ARCH),arm)
+	RUST_TRIPLE:=arm-unknown-linux-musleabihf
```

Comment:
going to need to keep this in mind when we add new devices.

Reply:
True, couldn't come up with anything better, but the triples are just different and implicit regex wizardry might not get us too far in this case.